metaserver/src/main/scala/scala/meta/languageserver/refactoring/OrganizeImports.scala

```diff
@@ -25,17 +26,24 @@ object OrganizeImports extends LazyLogging {
   def removeUnused(
       arguments: Option[Seq[JsValue]],
       index: SymbolIndex
-  ): ApplyWorkspaceEditParams = {
+  ): Task[Either[Response.Error, ApplyWorkspaceEditParams]] = {
     val result = for {
       as <- arguments
       argument <- as.headOption
       textDocument <- Json.fromJson[TextDocumentIdentifier](argument).asOpt
     } yield removeUnused(Uri(textDocument), index)
-    result.getOrElse(
-      throw InvalidParamsResponseError(
-        s"Unable to parse TextDocumentIdentifier from $arguments"
-      )
-    )
+    Task {
+      result match {
```

Comment:
😇 

```scala
import cats.syntax.either._

Either.fromOption(
  result,
  Left(
    Response.invalidParams(
      s"Unable to parse TextDocumentIdentifier from $arguments"
  )
)
```

Reply:
👍

Reply:
Nice! Only gripe with these APIs is finding the right imports, I just want to type `Either.fromOpt<TAB>` and have the editor insert the import. Only if we had a language server to do this for us 🤔